06-reactjs/components/movie-table.js

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+var MovieRow = require('./movie-row');
+
+var MovieTable = React.createClass({
+    propTypes: function () {
+        data: React.PropTypes.object;
+    },
+    render: function () {
+        var rows = [];
+         this.props.data.forEach (function (movie) {
```

Comment:
What happens if this.props.data is equal to undefined? Maybe you need to set data as required inside the propTypes or you can validate that this.props.data is defined when trying to do forEach